src/compiler/parser.ts

```diff
@@ -6468,7 +6468,7 @@ namespace ts {
 
                 function parseTagComments(indent: number) {
                     const comments: string[] = [];
-                    let state = JSDocState.SawAsterisk;

```

Comment:
Ah yes I should clarify. This isn't irrelevant. It fixes a bug I discovered while running the new quickinfo test I added. Consider this comment:

```ts
/**
  * @mytag1 
  * here all the comments are on a new line
  * @mytag2
  */
```

When the comment parser starts out at "SawAsterisk" state I get this tag comment for  mytag1: `"* here all the comments are on a new line"`

With my change it's fixed: `"here all the comments are on a new line"`

This is because we skip all the whitespace (including newlines) before we begin parsing so we start right at the *. This would have been handled in the AsteriskToken case below, but only if the last state had been BeginningOfLine. 

Going over the whole logic I believe it just makes more sense this way. Starting at BeginningOfLine is more logical since we didn't actually see an asterisk so why start at SawAsterisk...